common/lib/capa/capa/templates/choicetext.html

```diff
@@ -25,6 +25,8 @@
                     correctness = 'correct'
                 elif status == 'incorrect':
                     correctness = 'incorrect'
+                elif status == 'partially-correct':
+                    correctness = 'partialcorrect'   #Yes, this is different from how it's named elsewhere. :(
```

Comment:
Because the css class names are different from the correctness designation in the python, and I didn't want to change them in case someone was actually using them.